go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/rollkit/celestia-openrpc
 
-go 1.20
```

Comment:
Ah wait, this is going to cause issues for nitro, right @Ferret-san ?

Reply:
yes, unfortunately